scripts/test/mender-convert-qemu

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "$DISK_IMG" ]; then
+    echo "ERROR: DISK_IMG is not set"
+    exit 1
+fi
+
+# Try common locations for OVMF
+ovmf_file=""
+for maybe_ovmf_file in /usr/share/OVMF/OVMF_CODE.fd /usr/share/ovmf/x64/OVMF_CODE.fd; do
+    if [ -f $maybe_ovmf_file ]; then
+        ovmf_file=$maybe_ovmf_file
+    fi
+done
+
+if [ -z "${ovmf_file}" ]; then
+    echo "ERROR: could not find OVMF file"
+    exit 1
+fi
+
+qemu-system-x86_64 \
+    -enable-kvm \
+    -nographic \
+    -m 256 \
+    -net user,hostfwd=tcp::8822-:22 \
+    -net nic,macaddr=52:54:00$(od -txC -An -N3 /dev/urandom|tr \  :) \
+    -bios ${ovmf_file} \
+    -drive format=raw,file=${DISK_IMG} &
+
+qemu_pid=$!
+
+trap qemu_cleanup HUP KILL TERM
```

Comment:
should we also add INT and CHLD here?

Reply:
I add these to (almost) follow the [original script in meta-mender](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-qemu/scripts/mender-qemu#L20). And actually, we know that is only TERM signal being sent (test code [here](https://github.com/mendersoftware/meta-mender/pull/1145/files#diff-a0376fd2164bc856f75068a78c3eb3e26a5232e47bdaa1417da392b538cb540fL175)), so is it really important to add more?

Reply:
if we are sure they never get sent it is all right. for the same reasons no harm in adding them.